images/kubernetesonarm/elasticsearch/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM larmog/armhf-alpine-java:jdk-8u73
```

Comment:
I'm already working on a `luxas/java` image for this, will upload when I've got `libc` compiled for alpine.